docs/cmdline-opts/gen.pl

```diff
@@ -369,8 +369,8 @@ sub listhelp {
 
         my $line = sprintf "  {\"%s\",\n   \"%s\",\n   %s},\n", $opt, $desc, $bitmask;
 
-        if(length($opt) + length($desc) > 78) {
-            print STDERR "WARN: the --$long line is too long\n";
+        if(length($opt) > 78 || length($desc) > 78 || length($bitmask) > 78) {
+            print STDERR "WARN: some --$long lines are too long\n";
```

Comment:
I think this should spell out exactly what is too long in the error message. Maybe easiest by doing three different checks?

However, I don't think the bitmask length is the same as the others as that's a generated set of defines that this script could instead line-wrap if it turns out too long.

Reply:
I updated it as you wanted it to some degree and removed the bitmask length check.